### PR TITLE
Add more info to docs

### PR DESCRIPTION
**Pull request by javier-alvarez**
Add pre-requisites

### CODE REVIEW

**Comment by marrobi** [comment]
@javier-alvarez @deniscep feels like this should be in a different repository as is going to confuse things.

**Comment by christoferlof** [comment]
This feature branch is the outcome of #25 and will eventually be deleted as we incorporate AML etc into workspace services going forward.

File: InnerEyeAMLPLDeployment/README.md [hunk 1]
```diff
@@ -10,6 +10,15 @@ With some limitations listed below, we were able to successfully perform the fol
 
 ![AML in a closed VNet](./assets/PrivateAML.png)
 
+### Pre-Requisites
```
Comment (christoferlof) [inline InnerEyeAMLPLDeployment/README.md:13]: These are the steps to install the required tools on a Mac. Terraform, Azure CLI will be installed differently on Windows.

**Review by christoferlof — APPROVED** [review]

File: InnerEyeAMLPLDeployment/README.md (hunk 1 above)
Comment (christoferlof) [reply InnerEyeAMLPLDeployment/README.md:13]: @javier-alvarez , please update this heading or section to highlight that these are the steps on a Mac.  Also please check md linting error (don't use bare urls)

**Review by deniscep — APPROVED** [review]

**Pull request merged**